Data/sync/BraveCryptoJSFunctions.swift

```diff
@@ -0,0 +1,41 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
```

Comment:
I have a few comments on this class that may be super off, so if you feeling I'm missing the larger architecture stuff here, then let's chat more about it.

Reply:
The only problem there is having to manually copy and paste function declaration from javascript. No easy way around that, most we can do is to try to write at least one(happy path) unit test for each browserified script, so we are sure the function gets called